lightclient.ts

```diff
@@ -148,9 +148,9 @@ namespace jacdac {
 
         setStrip(numpixels: number, type = LightType.WS2812B_GRB, maxpower = 500): void {
             this._length = numpixels
-            this.setRegInt(JDLightReg.NumPixels, numpixels)
-            this.setRegInt(JDLightReg.LightType, type)
-            this.setRegInt(REG_MAX_POWER, maxpower)
+            this.setRegInt(LightReg.NumPixels, numpixels)
+            this.setRegInt(LightReg.LightType, type)
+            this.setRegInt(SystemReg.MaxPower, maxpower)
```

Comment:
is this an optional register for all services?

Reply:
No, it should be specified in every service